Amazon.IonDotnet/Internals/Text/TextScanner.cs

```diff
@@ -1051,6 +1051,10 @@ public int LoadDoubleQuotedString(StringBuilder sb, bool isClob)
                     {
                         sb.Append((char)c);
                         c = ReadChar();
+                        if ((char)c == '\\')
```

Comment:
Cannot we push this logic to line 1037?

Reply:
Removed this conditional as the ReadEscapeChar method does this check. Thanks.

Reply:
The switch condition is closed, and I don't think we should call the function again as it goes through other workflows